openlibrary/core/schema.sql

```diff
@@ -69,6 +69,7 @@ CREATE TABLE community_edits_queue (
     submitter text not null,
     reviewer text default null,
     url text not null,
+    type int not null,
```

Comment:
```suggestion
    type int not null default 1,
```
We should set a default here so that new MRs created in production have a `type`.